src/Build/BackEnd/Components/ProjectCache/ProjectCacheService.cs

```diff
@@ -525,7 +525,7 @@ private async Task<CacheResult> GetCacheResultAsync(BuildRequestData buildReques
 
                     ErrorUtilities.VerifyThrow(plugin.Instance != null, "Plugin '{0}' instance is null", plugin.Name);
 
-                    MSBuildEventSource.Log.ProjectCacheGetCacheResultStart(plugin.Name, buildRequest.ProjectFullPath, targetNames);
+                    MSBuildEventSource.Log.ProjectCacheGetCacheResultStart(plugin.Name, buildRequest.ProjectFullPath, targetNames ?? "<default>");
```

Comment:
Hello, sorry for jumping in,  asking out of curiosity: will it make sence to move the "<default>" to constants for possible future re-usage?

Reply:
In fact, there was a preexisting constant for this meaning that I should have used, thanks!